backend/src/test/java/com/google/rolecall/restcontrollers/UserManagementUnitTests.java

```diff
@@ -61,22 +64,26 @@ public void init() {
         mock(CastMemberRepository.class),null));
     controller = new UserManagement(userService);
     User.Builder builder = User.newBuilder()
-        .setFirstName(firstName)
-        .setLastName(lastName)
-        .setEmail(email)
-        .setDateJoined(dateJoined)
-        .setEmergencyContactName(emergencyContactName)
-        .setEmergencyContactNumber(emergencyContactNumber)
-        .setComments(comments)
-        .setIsActive(isActive)
-        .setCanLogin(canLogin)
-        .setAdmin(admin)
-        .setRecievesNotifications(notifications)
-        .setManagePerformances(managePerformances)
-        .setManageCasts(manageCasts)
-        .setManagePieces(managePieces)
-        .setManageRoles(manageRoles)
-        .setManageRules(manageRules);
+      .setFirstName(firstName)
+      .setLastName(lastName)
+      .setEmail(email)
+      .setDateJoined(dateJoined)
+      .setIsAdmin(isAdmin)
+      .setIsCoreographer(isAdmin)
+      .setIsDancer(isDancer)
+      .setIsOther(isOther)
+      .setCanLogin(canLogin)
+      .setRecievesNotifications(notifications)
+      .setManagePerformances(managePerformances)
+      .setManageCasts(manageCasts)
+      .setManagePieces(managePieces)
+      .setManageRoles(manageRoles)
+      .setManageRules(manageRules)
+      .setEmergencyContactName(emergencyContactName)
+      .setEmergencyContactNumber(emergencyContactNumber)
+      .setComments(comments)
+      .setIsActive(isActive)
+      ;
```

Comment:
```suggestion
      .setIsActive(isActive);
```

Reply:
See my comment above.

Reply:
Given that Mikhail also didn't like the trailing ; I am revering that change.